spec/models/timeline/monitoring_review_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe Timeline::MonitoringEvent do
+  let(:date) { 1.day.ago.strftime("%F") }
+  let(:review_id) { "Test-Review-ID" }
+  let(:issue_id) { "Test-Issue-ID" }
+  # just setting the start_date manually because we aren't triggering the before_validation callback
+  let(:event_param) { IssueMonitoringReview.new review_id: review_id, issue_id: issue_id, start_date: date }
+  let(:monitoring_review) { event_param.monitoring_review }
```

Comment:
[suggestion] this doesn't look used, can we axe it?

Reply:
Oooh I'll go check that out.  Maybe we can ditch the line I grabbed from the tta_review_spec, too

Reply:
TTA is using it but monitoring is not.  Nice catch!